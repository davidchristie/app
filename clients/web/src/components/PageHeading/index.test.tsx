import { PageHeading } from ".";
import { getByTestId, render, screen } from "../../testing";

describe("PageHeading", () => {
  it("renders correctly", () => {
    const { container } = render(
      <PageHeading>
        <div data-testid="children" />
      </PageHeading>
    );
    const element = screen.getByTestId("PageHeading");
    expect(getByTestId(element, "children")).toBeDefined();
    expect(element).toBe(container.firstChild);
    expect(element).toMatchSnapshot();
  });
});
