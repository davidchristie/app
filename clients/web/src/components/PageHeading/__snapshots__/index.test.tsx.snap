// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeading renders correctly 1`] = `
<h2
  class="MuiTypography-root MuiTypography-h2"
  data-testid="PageHeading"
>
  <div
    data-testid="children"
  />
</h2>
`;
